.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 lib
 es
+.idea
```

Comment:
Personally, i don't care but I know that some contributors will reject this one.